src/store/registration/index.ts

```diff
@@ -102,9 +101,11 @@ const slice = createSlice({
     },
     registerWithEmail: (state, _action: PayloadAction<null>) => {
       state.stage = RegistrationStage.EmailAccountCreation;
+      state.errors = [];
```

Comment:
required to clear errors if toggling from creating a web3 account to email or vice versa